web/doppelkopfSchreiben.html

```diff
@@ -21,6 +21,59 @@
       <th class="game">Spieler 2</th>
     </tr>
     </table>
+    <form class="write-blog" id="write-blog" method="post">
+      <div class="write-div">
+        <select id="winner1" class="write-input">
+        </select>
+
+        <select id="winner2" class="write-input">
+        </select>
+      </div>
+
+      <div class="write-div">
+        <div class="write-input">
+          <input type="radio" name="team" id="re" value="re" checked/>
+          <label for="re">Re-Partei</label>
+        </div>
+
+        <div class="write-input">
+          <input type="radio" name="team" id="kontra" value="kontra"/>
+          <label for="kontra">Kontra-Partei</label>
+        </div>
+      </div>
+
+      <div class="write-div">
+        <div class="write-input">
+          <input type="checkbox" id="re-bid" value="reBid"/>
+          <label for="reBid">Re angesagt</label>
```

Comment:
Das `for` Attribut vom `<label>` bezieht sich auf die `id` des `<input>`-tags, nicht auf `value`. Insofern ist die `for`-Verknüpfung dieses `<label>`s gerade ungültig. Generell wäre es eh besser hier kein camelCase zu nehmen, sondern lisp-case, da das in HTML und CSS für IDs, Attribute, Namen usw. üblich ist.